lib/constant.ts

```diff
@@ -0,0 +1,8 @@
+// Template table raw queries
+export const TemplateSelectTokenByFormIdQuery =
```

Comment:
I doubt these queries are used more than once in the code base. Would just use strings rather than adding an additional file and variables that need to be managed